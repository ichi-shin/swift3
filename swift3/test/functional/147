#!/bin/bash
# Copyright(c)2014 NTT corp. All Rights Reserved.
# DB1-1-8-5_8

. ./common

S3USER=tester


_s3_put /bucket

upload_id=$(_s3_post /bucket/object?uploads | \
    _xpath '/InitiateMultipartUploadResult/UploadId/text()')


_s3_delete /bucket/object?uploadId=${upload_id}

_sw_get /bucket/object -D - | _filter_curl

_sw_get /bucket+segments | grep object/${upload_id} | wc -l

