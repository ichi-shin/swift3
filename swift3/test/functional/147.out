QA output created by 147
> s3curl --id tester -- -X PUT http://SWIFT_HOST/bucket... 200
> s3curl --id tester -- -X POST http://SWIFT_HOST/bucket/object?uploads... 200
> s3curl --id tester -- -X DELETE http://SWIFT_HOST/bucket/object?uploadId=UPLOAD_ID... 204
> curl -X GET -D - http://SWIFT_HOST/v1/TENANT/bucket/object... 404
HTTP/1.1 404 Not Found
Content-Length: LENGTH
Content-Type: text/html; charset=UTF-8
Date: DATE
X-Trans-Id: TXID

<html><h1>Not Found</h1><p>The resource could not be found.</p></html>> curl -X GET http://SWIFT_HOST/v1/TENANT/bucket+segments... 204
0
